### PR TITLE
#1 add missing auth param when initializing httpClient

### LINKED ISSUE #1: Application Discovery Service - frontend

### PR DESCRIPTION
**Pull request by tuanlc**

### CODE REVIEW

**Review by ppppbn — DISMISSED** [review]

File: src/Client.js [hunk 1]
```diff
@@ -7,12 +7,12 @@ export default class Client {
    * @param {String} options.baseURL Base URL of your ESN server
    * @param {String} options.auth    Auth stretagy
    */
-  constructor({ baseURL } = {}) {
+  constructor({ baseURL, auth } = {}) {
```
Comment (ppppbn) [inline src/Client.js:10]: Should we have some validators for auth too?

Comment (itswillta) [reply src/Client.js:10]: Should we have tests to ensure this will not be missing ever again?

File: src/Client.js [hunk 2]
```diff
@@ -7,12 +7,12 @@ export default class Client {
    * @param {String} options.baseURL Base URL of your ESN server
    * @param {String} options.auth    Auth stretagy
```
Comment (itswillta) [inline src/Client.js]:
```suggestion
   * @param {String} options.auth    Auth strategy
```

File: src/Client.js (hunk 1 above)
Comment (tuanlc) [reply src/Client.js:10]: > Should we have some validators for auth too?

IMO, no need for now. DEV uses the library and if auth is wrong, 401 response will be appear

Comment (tuanlc) [reply src/Client.js:10]: Added test

**Comment by itswillta** [comment]
Nice, our first usage of `jest` ever :+1:.

**Review by itswillta — APPROVED** [review]

**Review by rezk2ll — APPROVED** [review]

**Pull request merged**